easybuild/easyconfigs/g/GUDHI/GUDHI-3.4.1-GCC-9.3.0.eb

```diff
@@ -0,0 +1,90 @@
+easyblock = 'CMakeMake'
+
+name = 'GUDHI'
+version = '3.4.1'
+
+homepage = 'https://gudhi.inria.fr/'
+description = """
+The GUDHI library is a generic open source C++ library, with a Python interface, for Topological Data Analysis (TDA) and Higher Dimensional Geometry Understanding.
+The library offers state-of-the-art data structures and algorithms to construct simplicial complexes and compute persistent homology.
+The library comes with data sets, demos, examples and test suites.
+"""
+
+toolchain = {'name': 'GCC', 'version': '9.3.0'}
```

Comment:
The tests are complaining because they cannot find easyconfigs for the dependencies in the `GCC` toolchain. You should consider instead using, for example, `GCCcore`.